package.json

```diff
@@ -106,20 +154,6 @@
     "webpack-merge": "^0.14.1",
     "webpack-validator": "^2.2.9"
   },
-  "dependencies": {
```

Comment:
@jhen0409 how would you explain difference between what should be in `devDependencies`, and what goes in `dependencies` in `./package.json'?    FWIU anything that gets packaged with an app goes in `./app/package.json`; everything else belongs in the root `./package.json`.     And as such, it makes sense to me that `./package.json` only has a `devDependencies`..